Tests/Client/Custom Assertions/AssertNetworkRequest.swift

```diff
@@ -9,36 +9,37 @@ import XCTest
 @testable import StreamChatClient
 
 
-/// Synchronously waits for a network request that matches its properties with the given parameters.
-///
-/// The function periodically checks the `RequestRecorderURLProtocol.recordedRequests` and If no request matches the
-/// given parameters within the `timeout` period, this assertion fails with the time-out error.
-///
-/// The values specified in the `headers`, `queryParameters` and `body` represents the mandatory subset of
-/// the values the request must have. A request is valid even when it contains additional parameters than
-/// the ones specified in these values.
-///
-/// - Parameters:
-///   - method: The HTTP method the request.
-///   - path: The `path` part of the request's URL.
-///   - headers: The headers required for the request.
-///   - queryParameters: The query parameters required for the request.
-///   - body: The expected body of the request.
-///   - timeout: The maximum time this function waits for a request to match the given parameters.
-///
-func AssertNetworkRequest(method: Endpoint.Method,
-                          path: String,
-                          headers: [String: String]?,
-                          queryParameters: [String: String]?,
-                          body: [String: Any]?,
-                          timeout: TimeInterval = 0.5,
-                          file: StaticString = #file,
-                          line: UInt = #line) {
-    
-    AssertAsync.willBeTrue(RequestRecorderURLProtocol.recordedRequests.contains {
-        $0.matches(method, path, headers, queryParameters, body)
-    }, message: "AssertNetworkRequest failed to find a matching request")
-    
+extension Assert {
```

Comment:
This should be `AssertAsync` imo

Reply:
hmm that's odd. Could you be reading and old version of my changes?
I changed that in this commit: https://github.com/GetStream/stream-chat-swift/pull/412/commits/807b438f40bbd434dd98bf22b53b3d1685197bad

Reply:
oh, probably :) All good then